test/Microsoft.Extensions.CommandLineUtils.Tests/DotNetMuxerTests.cs

```diff
@@ -15,6 +15,7 @@ public void FindsTheMuxer()
             var muxerPath = DotNetMuxer.MuxerPath;
             Assert.NotNull(muxerPath);
             Assert.True(File.Exists(muxerPath), "The file did not exist");
+            Assert.True(Path.IsPathRooted(muxerPath), "The path should be rooted");
```

Comment:
Will do. On second thought, it would be good to add this to the DotNetMuxer too. Process.MainModule will return the path to corehost in standalone mode ( which is typically renamed to MyAppName.exe). This is muxer, so we should fallback to "dotnet" on PATH in that case.